### PR TITLE
Label bracketsio for replacing adobebrackets

### PR DESCRIPTION
**Pull request by wakco**
Adobe stopped developing Brackets a year or two ago, [brackets.io](https://brackets.io) has continued development.

### CODE REVIEW

**Comment by scriptingosx** [comment]
The change like this will break any Installomator workflows that use the existing `brackets` label. 

Is there a reason for this?

**Comment by wakco** [comment]
I don't see how, the existing label for the discontinued Adobe version of Brackets is adobebrackets (in adobebrackets.sh), I checked that before I created this. And the only reason I didn't leave my file called brackets.sh was because the Adobe one wasn't (which you will note is what I originally called it), the current name reflects the website [brackets.io](https://brackets.io).

**Comment by scriptingosx** [comment]
ah, it was that renaming of the file that confused me. The misunderstanding was all mine. All good!

**Comment by Theile** [comment]
Should we include the name of the old label in this label, then?

**Comment by scriptingosx** [comment]
should we remove `adobebrackets` or redirect `adobebrackets` to be the same as `bracketsio`?

Is that what you are asking @Theile ?

**Comment by Theile** [comment]
Redirect `adobebrackets`, but I have no clue if we break something, but if the old software is not updated, we wouldn't need it in our list either, I guess.

**Comment by wakco** [comment]
> Redirect `adobebrackets`, but I have no clue if we break something, but if the old software is not updated, we wouldn't need it in our list either, I guess.

Done as requested.

File: fragments/labels/bracketsio.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+adobebrackets|bracketsio)
```
Comment (scriptingosx) [inline fragments/labels/bracketsio.sh:1]: can you change this to be in two lines:

```
adobebrackets|
bracketsio)
```

the other format breaks the algorithm to list the labels...

**Comment by scriptingosx** [comment]
running it right now returns an error:

```
2023-04-12 09:00:35 : REQ   : bracketsio : ################## Start Installomator v. 10.4beta, date 2023-04-12
2023-04-12 09:00:35 : INFO  : bracketsio : ################## Version: 10.4beta
2023-04-12 09:00:35 : INFO  : bracketsio : ################## Date: 2023-04-12
2023-04-12 09:00:35 : INFO  : bracketsio : ################## bracketsio
2023-04-12 09:00:35 : DEBUG : bracketsio : DEBUG mode 1 enabled.
2023-04-12 09:00:38 : DEBUG : bracketsio : name=Brackets
2023-04-12 09:00:38 : DEBUG : bracketsio : appName=
2023-04-12 09:00:38 : DEBUG : bracketsio : type=dmg
2023-04-12 09:00:38 : DEBUG : bracketsio : archiveName=
2023-04-12 09:00:38 : DEBUG : bracketsio : downloadURL=2023-04-12 09:00:36 : INFO  : bracketsio : GitHub API not returning URL, trying https://github.com/brackets-cont/brackets/releases/latest.
2023-04-12 09:00:38 : DEBUG : bracketsio : https://github.com
2023-04-12 09:00:38 : DEBUG : bracketsio : curlOptions=
2023-04-12 09:00:38 : DEBUG : bracketsio : appNewVersion=2.2.1
2023-04-12 09:00:38 : DEBUG : bracketsio : appCustomVersion function: Not defined
2023-04-12 09:00:38 : DEBUG : bracketsio : versionKey=CFBundleShortVersionString
2023-04-12 09:00:38 : DEBUG : bracketsio : packageID=
2023-04-12 09:00:38 : DEBUG : bracketsio : pkgName=
2023-04-12 09:00:38 : DEBUG : bracketsio : choiceChangesXML=
2023-04-12 09:00:38 : DEBUG : bracketsio : expectedTeamID=8F632A866K
2023-04-12 09:00:38 : DEBUG : bracketsio : blockingProcesses=
2023-04-12 09:00:38 : DEBUG : bracketsio : installerTool=
2023-04-12 09:00:38 : DEBUG : bracketsio : CLIInstaller=
2023-04-12 09:00:38 : DEBUG : bracketsio : CLIArguments=
2023-04-12 09:00:38 : DEBUG : bracketsio : updateTool=
2023-04-12 09:00:38 : DEBUG : bracketsio : updateToolArguments=
2023-04-12 09:00:38 : DEBUG : bracketsio : updateToolRunAsCurrentUser=
2023-04-12 09:00:38 : INFO  : bracketsio : BLOCKING_PROCESS_ACTION=tell_user
2023-04-12 09:00:38 : INFO  : bracketsio : NOTIFY=success
2023-04-12 09:00:38 : INFO  : bracketsio : LOGGING=DEBUG
2023-04-12 09:00:38 : INFO  : bracketsio : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2023-04-12 09:00:38 : INFO  : bracketsio : Label type: dmg
2023-04-12 09:00:38 : INFO  : bracketsio : archiveName: Brackets.dmg
2023-04-12 09:00:38 : INFO  : bracketsio : no blocking processes defined, using Brackets as default
2023-04-12 09:00:38 : DEBUG : bracketsio : Changing directory to /Users/armin/BTSync/Work/Projects/Installomator/build
2023-04-12 09:00:38 : INFO  : bracketsio : name: Brackets, appName: Brackets.app
2023-04-12 09:00:38.405 mdfind[88403:5147039] [UserQueryParser] Loading keywords and predicates for locale "en_US"
2023-04-12 09:00:38.405 mdfind[88403:5147039] [UserQueryParser] Loading keywords and predicates for locale "en"
2023-04-12 09:00:38.443 mdfind[88403:5147039] Couldn't determine the mapping between prefab keywords and predicates.
2023-04-12 09:00:38 : WARN  : bracketsio : No previous app found
2023-04-12 09:00:38 : WARN  : bracketsio : could not find Brackets.app
2023-04-12 09:00:38 : INFO  : bracketsio : appversion: 
2023-04-12 09:00:38 : INFO  : bracketsio : Latest version of Brackets is 2.2.1
2023-04-12 09:00:38 : REQ   : bracketsio : Downloading 2023-04-12 09:00:36 : INFO  : bracketsio : GitHub API not returning URL, trying https://github.com/brackets-cont/brackets/releases/latest.
2023-04-12 09:00:38 : REQ   : bracketsio : https://github.com to Brackets.dmg
2023-04-12 09:00:38 : DEBUG : bracketsio : No Dialog connection, just download
2023-04-12 09:00:38 : ERROR : bracketsio : error downloading 2023-04-12 09:00:36 : INFO  : bracketsio : GitHub API not returning URL, trying https://github.com/brackets-cont/brackets/releases/latest.
2023-04-12 09:00:38 : ERROR : bracketsio : https://github.com
ls: Brackets.dmg: No such file or directory
2023-04-12 09:00:38 : ERROR : bracketsio : File list: 
2023-04-12 09:00:38 : ERROR : bracketsio : File type: Brackets.dmg: cannot open `Brackets.dmg' (No such file or directory)
2023-04-12 09:00:38 : DEBUG : bracketsio : DEBUG mode 1, not reopening anything
2023-04-12 09:00:38 : ERROR : bracketsio : ERROR: Error downloading 2023-04-12 09:00:36 : INFO  : bracketsio : GitHub API not returning URL, trying https://github.com/brackets-cont/brackets/releases/latest.
2023-04-12 09:00:38 : ERROR : bracketsio : https://github.com error:
* Closing connection -1
curl: (3) URL using bad/illegal format or missing URL

2023-04-12 09:00:38 : REQ   : bracketsio : ################## End Installomator, exit code 2 
```

The reason for this error is that the project's latest release is a "windows only patch" and does not have a dmg attached 🤦

File: fragments/labels/bracketsio.sh (hunk 1 above)
Comment (wakco) [reply fragments/labels/bracketsio.sh]: Done, any suggestions on how to resolve the windows only update issue?

**Comment by scriptingosx** [comment]
That would require replacing the downloadURLFromGit function with something custom (or better, fixing the downloadURLFromGit to be a bit resilient here...)

For now, I will keep this open until there is a new release and then hope that don't frequently do single platform patches...

**Comment by scriptingosx** [comment]
Download works again. Team ID changed in the interim. I fixed it in the merge. Thank you

**Pull request merged**